src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -16,10 +16,7 @@
 
 from hyp3_isce2 import packaging, topsapp
 from hyp3_isce2.burst import (
-    download_bursts,
-    get_burst_params,
```

Comment:
Same with this function. You'll also need to update [download_example.py](https://github.com/ASFHyP3/hyp3-isce2/blob/develop/src/hyp3_isce2/etc/download_example.py) (to remove now non-existent functions, and update the [product readme](https://github.com/ASFHyP3/hyp3-isce2/blob/3e4fd6f0c7d85d6fdd33ea152f3439af80b05ff7/src/hyp3_isce2/metadata/templates/insar_burst/insar_burst_readme.md.txt.j2#L73) to match the new functionality.